app/frontend/src/features/messages/groupchats/GroupChatsTab.stories.tsx

```diff
@@ -12,7 +12,7 @@ import GroupChatView from "./GroupChatView";
 const queryClient = new QueryClient();
 
 export default {
-  title: "GroupChatsTab",
+  title: "Messages/GroupChatsTab",
```

Comment:
The chat page in this is also broken... I wonder if it's the mocks playing up...

Reply:
It was because it had changed from being passed the chat, to querying it using the id in the url